frontends/ol-utilities/src/ssr/NoSSR.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { useState, useEffect, ReactNode } from "react"
+
+type NoSSRProps = {
+  children: ReactNode
+  onSSR?: ReactNode
+}
+
+export const NoSSR: React.FC<NoSSRProps> = ({ children, onSSR = <></> }) => {
+  const [isClient, setClient] = useState(false)
+
+  useEffect(() => {
+    setClient(true)
+  }, [])
+
+  return <>{isClient ? children : onSSR}</>
```

Comment:
Any need for the fragment wrapper?

And maybe just `null` for `onSSR` default?